### PR TITLE
PUBDEV-7217: Add information about print_mojo to Productionizing chapter; add print_mojo example to Python docs

### PR DESCRIPTION
**Pull request by hannah-tillman**

### CODE REVIEW

File: h2o-docs/src/product/mojo-quickstart.rst [hunk 1]
```diff
@@ -693,6 +693,54 @@ If you cannot install Graphviz on your environment, another option is to produce
   :alt: Example Output Picture
   :scale: 50%
 
+GBM MOJO to ONNX Converter
```
Comment (honzasterba) [inline h2o-docs/src/product/mojo-quickstart.rst:696]: this is not accurate, the json output is just a intermediate format which is then used by ONNXMLTools to generate the final ONNX model.

https://github.com/onnx/onnxmltools#h2o-to-onnx-conversion

Comment (angela0xdata) [reply h2o-docs/src/product/mojo-quickstart.rst:696]: @hannah-tillman just change the heading to "Printing GBM MOJOs"

File: h2o-docs/src/product/mojo-quickstart.rst [hunk 2]
```diff
@@ -693,6 +693,54 @@ If you cannot install Graphviz on your environment, another option is to produce
   :alt: Example Output Picture
   :scale: 50%
 
+Printing GBM MOJOs
+~~~~~~~~~~~~~~~~~~
+
+The ``print_mojo`` function allows a GBM model to be coverted to `ONNX <https://onnx.ai/about.html>`__ representation, which means the model will be rendered in an "open format built to represent" ML models. This allows you to use the model with a variety of frameworks, tools, runtimes, and compilers.
```
Comment (honzasterba) [inline h2o-docs/src/product/mojo-quickstart.rst:699]: this is still innacurate, the output has nothing to do with ONNX, also this works for all tree-like models

I would reduce this to

"The ``print_mojo`` function allows a model to be converted to machine readable representation - either to JSON for further processing, dot for rendering images.

@koniecsveta maybe we should update print_mojofunction to also allow PNG output?

Comment (koniecsveta) [reply h2o-docs/src/product/mojo-quickstart.rst]: yesyes, I think it will be an easy thing after adding it to flow.

**Pull request merged**